package.json

```diff
@@ -42,15 +42,15 @@
     "stylelint-config-standard": "^22.0.0",
     "ts-node-dev": "^1.1.8",
     "typesafe-actions": "^5.1.0",
-    "typescript": "^4.3.2",
-    "vite": "^2.4.4"
+    "typescript": "^4.3.2"
   },
   "dependencies": {
     "dotenv": "^10.0.0",
     "express": "^4.17.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router-dom": "^6.0.1",
-    "styled-components": "^5.3.3"
+    "styled-components": "^5.3.3",
+    "vite": "^2.6.14"
```

Comment:
When you re-installed `vite` you forgot to install it as a dev-dependency, you can do that with the `-D` option for `npm install`. `vite` is a dependency which is not needed, outside the development process. 
[Read more here](https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file)

Reply:
perfect! thank you very much